winfetch.ps1

```diff
@@ -178,6 +218,16 @@ $defaultConfig = @'
 # disabling unused ones will improve speed
 # $ShowPkgs = @("scoop", "choco")
 
+# Configure how to show info for levels
+# 0 is for text only. (Default)
+# 1 is for bar only.
+# 2 is for text + bar.
+# 3 is for bar + text.
+# $cpustyle = 1
+# $memorystyle = 2
+# $diskstyle = 3
+# $batterystyle = 3
+
```

Comment:
We can use strings here and in the switch on line 135, that'd be much easier for users and wouldn't require as much explanation.
e.g. $cpustyle = "infobar"